tests/installation/partitioning_olddisk.pm

```diff
@@ -0,0 +1,96 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+#
+# Summary:  Install on old disk
+# Maintainer: Joyce Na <jna@suse.de>
+
+use strict;
+use warnings;
+use base "y2logsstep";
+use version_utils qw(is_storage_ng is_sle);
+use testapi;
+
+sub run() {
+    send_key $cmd{expertpartitioner};
+    #assert_screen 'expert-partitioner-setup';
+    if (is_storage_ng && is_sle('15+')) {
+        # start with preconfigured partitions
+        send_key 'down';
+        send_key 'ret';
+    }
+    assert_screen 'expert-partitioner-setup';
+    sleep 3;
+    # Rescan devices
+    if (is_storage_ng && is_sle('15+')) {
+        send_key $cmd{rescandevices};
+        assert_screen 'rescan-devices-warning';    # Confirm rescan
+        send_key 'alt-y';
+    }
+    else {
+        send_key 'alt-e';
+    }
+    wait_still_screen;                             # Wait until rescan is done
+
+    # Select device
+    my $disk = get_var('SPECIFICDISK');
+    send_key 'tab';
+    send_key 'tab';
+    send_key_until_needlematch("expert-partitioner-$disk", "down", 20, 2);
+    send_key 'ret';
+
+    # Edit device
+    send_key 'alt-e';
+    assert_screen 'expert-partitioner-sda-edit';
+
+    # Format partition with brtfs file system
+    send_key 'alt-a';
+    assert_screen 'expert-partitioner-format';
+    send_key 'ret';
+    if (is_storage_ng && is_sle('15+')) {
+        send_key 'alt-f';
+    }
+    else {
+        send_key 'alt-s';
+    }
+    send_key_until_needlematch("expert-partitioner-brtfs", "up", 20, 1);
+    send_key 'ret';
+
+    # Set mount point and volume label
+    send_key 'alt-o';
+    send_key 'alt-m';
+    for (1 .. 5) { send_key "backspace" }
+    send_key '/';
```

Comment:
I guess we 80% of these already implemented. Could you please take a look in `lib/partition_setup.pm` and it's helper functions?

Reply:
the build need install on specific partition,so before set mount point,it need delete the original mount point, I've checked sub mount_device in lib/partition_setup.pm, it can not delete the original setting,so I add this.

Reply:
that's the point, I guess this use-case can be useful for others as well, so instead of keeping it hidden I propose extracting it to the library, so it can be reused. As of now that library has focus on new setups, so if will do partitioning from scratch by wiping existing partitions, you can just reuse those. Also, the only missing thing in `mount_device` is cleaning up the textbox, which makes sense for all scenarios, so feel free to add it there. However, it should be more scalable then just 5 backspace key presses, as we might have more symbols there.

Reply:
Updated,and moved this part to lib/partition_sestup.pm mount_device